docs/release-notes/release-notes-0.19.3.md

```diff
@@ -29,6 +29,13 @@
 
 ## Functional Enhancements
 
+- Previously, when sweeping non-time sensitive anchor outputs, they might be
+  grouped with other non-time sensitive outputs such as `to_local` outputs,
+  which potentially allow the sweeping tx to be pinned. This is now
+  [fixed](https://github.com/lightningnetwork/lnd/pull/10117) by moving sweeping
+  anchors into its own tx, which means the anchor outputs won't be swept in a
```

Comment:
would be cool to actually group at least anchors together, seems like sub 1 sat/vbyte transactions can now be propagated quite reliably

Reply:
Gotta say I'm very confused, as on one hand you were saying you really wanna the anchor sweeping to be removed but now you think it's cool to group them?

Reply:
I was gonna create a PR to remove the anchor sweeping - based on your current view, it seems unnecessary to create that PR?

Reply:
Curious what your opinion is, but I would say either we remove the anchors or we make sure we can group different anchors from different channel force closes and make also sure we can broadcast them sub 1 sat/vbyte ? Current version as mentioned before never sweeps the anchors anyways so I think we need an improvement anyways here going forward ?